### PR TITLE
fix: filter type for item query

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
**Issue**
Screenshot 2021-06-03 at 5 57 37 PM

### CODE REVIEW

**Comment by barredterra** [comment]
Thanks for fixing, @rohitwaghchaure !

File: erpnext/controllers/queries.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 from __future__ import unicode_literals
 import frappe
 import erpnext
+import json
+from six import string_types
```
Comment (ankush) [inline erpnext/controllers/queries.py]:
```suggestion
```

File: erpnext/controllers/queries.py [hunk 2]
```diff
@@ -198,6 +200,9 @@ def get_accounts(with_account_type_filter):
 def item_query(doctype, txt, searchfield, start, page_len, filters, as_dict=False):
 	conditions = []
 
+	if isinstance(filters, string_types):
```
Comment (ankush) [inline erpnext/controllers/queries.py]:
```suggestion
	if isinstance(filters, str):
```

**Review by ankush — COMMENTED** [review]
We can ignore py2 support now.

**Review by barredterra — APPROVED** [review]

**Pull request merged**